src/java/org/apache/cassandra/cql3/functions/VectorFcts.java

```diff
@@ -33,9 +33,11 @@
 import org.apache.cassandra.db.marshal.NumberType;
 import org.apache.cassandra.db.marshal.VectorType;
 import org.apache.cassandra.exceptions.InvalidRequestException;
+import org.apache.cassandra.index.sai.disk.vector.VectorValidation;
 import org.apache.cassandra.transport.ProtocolVersion;
 
 import static java.lang.String.format;
+import static org.apache.cassandra.index.sai.disk.vector.VectorValidation.MAX_FLOAT32_COMPONENT;
```

Comment:
Would it make sense to put this constant in `VectorType`, so we keep CQL independent of SAI, which is a particular index implementation? This kind of dependencies have been problematic during SAI porting efforts.

Reply:
Makes sense, +1